docs/application/vscode-ext/web.md

```diff
@@ -333,46 +333,103 @@ To debug application, follow these steps:
+    To launch Design Editor, click **Design-Editor** on the VS Code window as shown in the following figure:
 
     ![Design Editor button](media/tizen_web27.png)
-- Right-click preference
-    1. Select the HTML file that you want to launch and right-click.
+
+- Using right-click options
+    1. To launch Design Editor, select the HTML file that you want to launch and right-click.
     2. From the list that appears, click **Tizen Web: Run Design-Editor**.
 
         ![Design Editor right click](media/tizen_web28.png)
 
-Design Editor is successfully launched.
+### Features
 
-![Design Editor launch](media/tizen_web29.png)
+Following are the WYSIWYG Design Editor features:
 
-### Quick Action Execution
-Right-click on the **project folder** to use the extension function in the menu.
+- Drag and Drop a component
 
-For more information on each function, see:
-- [Tizen Web: Build Tizen App](#Build-Project)
-- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
-- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
-- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
-- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+    You can drag and drop a component from the Widget palette to the canvas.
+
+    ![Design Editor Widget](media/tizen_web35.png)
+    
+- Change a component properties
+
+    After placing the UI components, you can set their properties in the Attribute palette.
+
+    ![Design Editor Attribute](media/tizen_web36.png)
+
+- Preview
+
+    You can use **Preview** ![Design Editor PreviewIcon](media/tizen_web45.png) in the toolbar to display the preview of the edited page.
+
+    ![Design Editor Preview](media/tizen_web37.png)
+   
+- JavaScript Assistant
+
+   You can use **JavaScript Assistant** ![Design Editor JSAssitantIcon](media/tizen_web46.png) in the toolbar to edit the Javascript code with **JavaScript Assistant Wizard**.
+
+    ![Design Editor JavaScript Assistant](media/tizen_web38.png)
+
+- Structure View
+
+   You can use **Structure View** ![Design Editor StructureIcon](media/tizen_web47.png) in the toolbar, to see the DOM tree of the page.
+
+   ![Design Editor Structure](media/tizen_web39.png)
+
+- New Page 
+
+   You can use **New Page** ![Design Editor NewPageIcon](media/tizen_web48.png) in the toolbar to add a new page with empty, header, footer, or list template.
+
+   ![Design Editor Newpage](media/tizen_web40.png)
+
+- HTML Assistant
+
+   You can use **HTML Assistant** ![Design Editor HTMLAssistantIcon](media/tizen_web49.png) in the toolbar to edit the HTML code.
+
+   ![Design Editor HTML Assistant](media/tizen_web41.png)
+
+- Undo or Redo 
+
+   You can use **Undo** ![Design Editor UndoIcon](media/tizen_web50.png) or **Redo** ![Design Editor RedoIcon](media/tizen_web51.png) in the toolbar to undo or redo any action.
+
+### Usage of Design Editor 
+
+- Usage of Radio button with HTML Assistant
```

Comment:
Suggest adding a line space. 
In the view tab, the image is immediately after the bulleted list.
Please verify.